components/dropdown/index.js

```diff
@@ -0,0 +1,82 @@
+import { PATHS } from 'constants.js';
+import React from 'react';
+import colors from '../../color/color.json';
+import Link from 'next/link';
+import signOut from 'utils/signOut';
+
+const Dropdown = () => {
+  return (
+    <div className="dropdown">
+      <ul className="dropdownList">
+        <li className="dropdownItem">
+          <Link href={PATHS.PROFILE}>
+            <p className="profileLink">My Profile</p>
+          </Link>
+        </li>
+        <hr className="line" />
+        <li
+          onClick={() => {
+            signOut();
+          }}
```

Comment:
Why do we need a callback here? It can be like `onClick={signOut}`

Reply:
That's a better way 💯 , Will update 👍🏻